generator/config/stage/fill.yaml

```diff
@@ -40,3 +40,71 @@ arguments:
         description: |
             Specifies an object containing each field for which to fill missing values. You can specify multiple fields in the output object.
             The object name is the name of the field to fill. The object value specifies how the field is filled.
+tests:
+    -
+        name: 'Fill Missing Field Values with a Constant Value'
+        link: 'https://www.mongodb.com/docs/manual/reference/operator/aggregation/fill/#fill-missing-field-values-with-a-constant-value'
+        pipeline:
+            -
+                $fill:
+                    output:
+                        bootsSold:
+                            value: 0
+                        sandalsSold:
+                            value: 0
+                        sneakersSold:
+                            value: 0
+    -
+        name: 'Fill Missing Field Values with Linear Interpolation'
+        link: 'https://www.mongodb.com/docs/manual/reference/operator/aggregation/fill/#fill-missing-field-values-with-linear-interpolation'
+        pipeline:
+            -
+                $fill:
+                    sortBy:
+                        time: 1
+                    output:
+                        price:
+                            method: 'linear'
```

Comment:
We an create a factory for the output subtype.

Reply:
@alcaeus: Correct me if I'm wrong, but that's basically what you did for ODM already, yes?